### DISCUSSION TITLE
how to change the index of the default color from the default palette, I want to change the color to 5 instead of 6

### DISCUSSION THREAD

**Discussion by m1x-developer** [mantinedev/mantine#6311] [Q&A]
how to change the index of the default color from the default palette, I want to change the color to 5 instead of 6

<img width="915" alt="image" src="https://github.com/mantinedev/mantine/assets/83704092/7974365e-7baa-4b5f-89d4-64803f589225">

**Comment by rtivital** [comment]
https://mantine.dev/theming/colors/#primaryshade
